### PR TITLE
Update documentation for the context manager and tag a new release

### PR DESCRIPTION
**Pull request by g--**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,3 +63,86 @@ def perform_request(self, req)
 
 When using decorators, metrics are only tracked if the decorated function
 does not raise an error.
+
+### Context
+
+Statscor includes a context manager that can help measure latency and volume
```
Comment (markstory) [inline README.md]:
```suggestion
Statsdecor includes a context manager that can help measure latency and volume
```

File: statsdecor/context.py [hunk 2]
```diff
@@ -8,26 +8,32 @@
...
-                elif exc_val is not None:
-                    self.add_tags('result', 'exception')
-                else:
-                    self.add_tags('result', 'success')
-
-        def call_thing():
-            with ThingyStatsContext() as stats:
-                result = thingy_external_call()
-                stats.add_tags('status_code', 'result["status_code"]')
+        A context used for timing and tagging code.  On entering the context,
```
Comment (markstory) [inline statsdecor/context.py]:
```suggestion
        A context manager used for timing and tagging code.  On entering the context,
```

File: README.md [hunk 3]
```diff
@@ -63,3 +63,86 @@ def perform_request(self, req)
...
+based on StatsContext:
+
+```
+from statsdecor.context import StatsContext
+
+class FoobarClientMetrics(StatsContext):
+    def __init__(self, tags=None):
+        tags = list(tags or [])
+        tags += ['caller:example_1']
+        super(ThingyStatsContext, self).__self__('thingy_client', tags=tags)
```
Comment (amcintosh) [inline README.md]: `super(FoobarClientMetrics, self)`? or just `super()`?

Comment (g--) [reply README.md]: I think statsdecor is python2 compatible.  But I think `super()` by itself would work if it's python3.  (yay!!! Finally!!)

**Pull request merged**